fuse_models/include/fuse_models/parameters/parameter_base.h

```diff
@@ -64,24 +63,14 @@ struct ParameterBase
 };
 
 /**
- * @brief Utility method for handling required ROS params
```

Comment:
The changes here are simply the two free functions (re)moved to `fuse_core/parameter.h`.